### ISSUE TITLE
[ `README` ] : Remove `CodeClimate`

### ISSUE THREAD

**Issue by baseplate-admin** [baseplate-admin/CoreProject#309]
Its been a while since i didn't use codeclimate in this project. So this means effectively we dont need to add codeclimate badge in the `README.md` file. So in my opinion we should remove the codeclimate badge :)

**Comment by baseplate-admin** [comment]
@AKindWorld What do you think ?

**Comment by baseplate-admin** [comment]
Remove the lines 

https://github.com/baseplate-admin/CoreProject/blob/9255a979e4974ea413570a19b982020d7b203653/README.md?plain=1#L37-L39

**Comment by baseplate-admin** [comment]
@AKindWorld You willing to submit a PR ? 📦

**Comment by AKindWorld** [comment]
> You willing to submit a PR ? 📦

Yup sure. I'll update the Readme a bit too with more (& updated) screenshots + the `Built with` portion. So, yup, this is in the to-do too. 

Let's keep this issue open till then, if you don't mind.

**Comment by baseplate-admin** [comment]
Okay :D But lets keep the format. Like the Master is protected. Dont commit against master :P Open a new branch and then submit a Pull Request. ( It would be better to do it in your fork ) then submit a direct PR

**Comment by baseplate-admin** [comment]
> Yup sure. I'll update the Readme a bit too with more (& updated) screenshots + the Built with portion. So, yup, this is in the to-do too.

You should delete your fork and refork this project
